src/module.ts

```diff
@@ -545,6 +557,17 @@ export default defineNuxtModule<ModuleOptions>({
         ws.broadcast({ event, key })
       })
     })
+
+    if (options.integrations) {
+      // @nuxtjs/tailwindcss support
+      if (options.integrations.tailwindcss) {
```

Comment:
What do you think about removing this option and just using the hook? The hook will not activate unless the user uses tailwincss.

Reply:
Resolved in [8e568c2](https://github.com/nuxt/content/pull/1351/commits/8e568c2a2cd371f92e7d8e27886127a0231cb193)